srtcore/socketconfig.h

```diff
@@ -915,7 +924,7 @@ struct CSrtConfigSetter<SRTO_PAYLOADSIZE>
             }
 
             size_t efc_max_payload_size = SRT_LIVE_MAX_PLSIZE - fc.extra_size;
-            if (co.zExpPayloadSize > efc_max_payload_size)
+            if (val > efc_max_payload_size)
```

Comment:
@ethouris Please confirm this fix in case I misunderstand the logic.

Reply:
No, you are not changing any logics here, it's ok.